packages/extension/src/components/atoms/Logo/Logo.tsx

```diff
@@ -1,29 +1,17 @@
 import { FC, SVGProps, useMemo } from 'react';
 
-import { useFeatureFlags } from '../../../hooks';
```

Comment:
The whole hook can be removed as it's not used anymore

Reply:
I would keep it as we might need it in the future